src/applications/vaos/tests/appointment-list/components/AppointmentsPage/AppointmentsPageV2.unit.spec.jsx

```diff
@@ -437,4 +442,154 @@ describe('VAOS <AppointmentsPageV2>', () => {
     ).to.be.ok;
     expect(screen.getByRole('link', { name: 'Request appointment' }));
   });
+
+  it('should navigate to confirmed appointments detail page', async () => {
+    // VA appointment id from confirmed_va.json
+    const url = '/va/var21cdc6741c00ac67b6cbf6b972d084c1';
+
+    const appointment = getVAAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      startDate: moment().format(),
+      clinicId: '308',
+      clinicFriendlyName: "Jennie's Lab",
+      facilityId: '983',
+      sta6aid: '983GC',
+      communityCare: false,
+      vdsAppointments: [
+        {
+          bookingNote: 'New issue: ASAP',
+          appointmentLength: '60',
+          appointmentTime: '2021-12-07T16:00:00Z',
+          clinic: {
+            name: 'CHY OPT VAR1',
+            askForCheckIn: false,
+            facilityCode: '983',
+          },
+          type: 'REGULAR',
+          currentStatus: 'NO ACTION TAKEN/TODAY',
+        },
+      ],
+      vvsAppointments: [],
+    };
+
+    mockAppointmentInfo({
+      va: [appointment],
+      cc: [],
+      requests: [],
+      isHomepageRefresh: true,
+    });
+
+    const facility = {
+      id: 'vha_442GC',
+      attributes: {
+        ...getVAFacilityMock().attributes,
+        type: 'facility',
+        address: {
+          mailing: {},
+          physical: {
+            zip: '80526-8108',
+            city: 'Fort Collins',
+            state: 'CO',
+            address1: '2509 Research Boulevard',
+            address2: null,
+            address3: null,
+          },
+        },
+        id: 'vha_442GC',
+        name: 'Fort Collins VA Clinic',
+        phone: {
+          fax: '970-407-7440',
+          main: '970-224-1550',
+          pharmacy: '866-420-6337',
+          afterHours: '307-778-7550',
+          patientAdvocate: '307-778-7550 x7517',
+          mentalHealthClinic: '307-778-7349',
+          enrollmentCoordinator: '307-778-7550 x7579',
+        },
+        uniqueId: '442GC',
+      },
+    };
+    mockFacilitiesFetch('vha_442GC', [facility]);
+
+    const screen = renderWithStoreAndRouter(
+      <MemoryRouter initialEntries={['/', url]}>
```

Comment:
There's already a Router component rendered in the `renderWithStoreAndRouter`. I think you could remove `MemoryRouter` here and possibly render `AppointmentListSection` from `src/applications/vaos/appointment-list/index.jsx`, so you don't have to put the individual routes in here. Mocking up specific routes like this makes the test easier to break later if we change how the routing works.

Reply:
I figured there had to be a better way to do this. I just didn't know how. I need a tutorial on how to used renderWithStoreAndRouter and renderFromRoutes.

Reply:
Done.